falco.yaml

```diff
@@ -982,6 +982,7 @@ metrics:
   libbpf_stats_enabled: true
   convert_memory_to_mb: true
   include_empty_values: false
+  prometheus_enabled: false
```

Comment:
Given that this is indeed exposed by the webserver, shouldn't it live under `webserver:`; perhaps we might want to specify that it is useful only when `metrics` are enabled.

Reply:
Good point and agreed 

One related question: should the endpoint name also be configurable ? It usually is `/metrics` but...

Reply:
I'd leave `/metrics` for now. We can always make int configurable in the future if there is demand for it!